dev-server/ui-playground/PlaygroundApp.js

```diff
@@ -1,44 +1,10 @@
 import { SvgIcon } from '@hypothesis/frontend-shared';
 
-import Button from '../../src/sidebar/components/Button';
-import Menu from '../../src/sidebar/components/Menu';
-import MenuItem from '../../src/sidebar/components/MenuItem';
+import ButtonsSharedDemo from './ButtonsSharedDemo';
```

Comment:
Just, again, early stages in the PlaygroundApp, so various refactors and changes...